src/sentry/api/endpoints/project_event_details.py

```diff
@@ -54,7 +54,7 @@ def wrap_event_response(
 class ProjectEventDetailsEndpoint(ProjectEndpoint):
     owner = ApiOwner.ISSUES
     publish_status = {
-        "GET": ApiPublishStatus.UNKNOWN,
+        "GET": ApiPublishStatus.PUBLIC,
```

Comment:
We shouldn't make APIs public before they're nicely documented. Public will make them available to in public api docs which is why the tests are failing. Instead please keep them `unknown` and create a task [here](https://www.notion.so/sentry/2fc7a7d2f2bf498f95e835492810b604?v=add3c94754cd4b99ba687211d9126292) with timeline of when documentation can be done. 

Here's documentation about how you can publish APIs: https://www.notion.so/sentry/2fc7a7d2f2bf498f95e835492810b604?v=add3c94754cd4b99ba687211d9126292